api/user.json

```diff
@@ -78,16 +96,53 @@
   ],
   "deductions": {
     "disabilityClaim": null,
+    "disabilityAmount": {
+      "name": "Disability",
+      "line": 316,
+      "amount": 0.00
+    },
     "rrspClaim": null,
+    "basicPersonalAmount": {
+      "name": "Basic Personal Amount",
+      "line": 300,
+      "amount": 11809.00
+    },
+    "ageAmount": {
+      "name": "Age Amount",
+      "line": 301,
+      "amount": 7500.00
+    },
     "rrspAmount": 0,
     "partnerAgeClaim": null,
     "canadaCaregiverClaim": null,
     "partnerDisabilityClaim": null,
     "partnerPensionIncomeClaim": null,
+    "pensionIncomeClaim": null,
+    "pensionIncomeClaimAmount": {
+      "name": "Pension",
+      "line": 314,
+      "amount": 700.00
+    },
     "childcareClaim": null,
+    "childcareAmount": {
+      "name": "Childcare Expenses",
+      "line": 214,
+      "amount": 1517.94
+    },
     "singleParentClaim": null,
     "dependentDisabilityClaim": null,
     "medicalExpenseClaim": null,
+    "medicalExpenseClaimAmount": {
+      "name": "Medical Expenses",
+      "line": 331,
+      "multiLine": "330/331",
+      "amount": 1227.34
+    },
+    "amountsFromSpouse": {
+      "name": "Amounts transfered from spouse",
+      "line": 326,
+      "amount": 0.00
```

Comment:
Do we want to show these even if they are `0`?

I understand this will wreak havoc on your `.space-row` logic, but let's double check with Brian what we should do with these in the future.

If it's too much work to actually hide them + keep the grouping in order, we can leave them all in for now.